### ISSUE TITLE
Vox players dont spawn in ERT gear

### ISSUE THREAD

**Issue by Xantholne** [OracleStation/OracleStation#737]
Issue reported from Round ID: 1327 (Oracle Station | Medium RP)

If you select yes and spawn in as a ERT whilst playing as a vox you do not get auto equipped with your hardsuit and other gear.

I 100% suspect this is because we disallow vox from wearing all types of hardsuits

**Comment by SmokingKilz** [comment]
Elaborate this please, you mean while ghosted, yes?

**Comment by Ralta** [comment]
Er,  I thought only humans could be in an ERT...

**Comment by Xantholne** [comment]
If Player Greytide McSkrek is a vox player that is currently a ghost
And admin sends ERT prompt
Player Greytide McSkrek spawns in however from how it seems to function is it doenst create the mob and equipment then put the gear on. It seems to create you as a mob and put the gear on but for whatever reason if you were a vox it does not equip you because of voxes incapabilities to equip ERT gear
Thusly it makes it so said vox player spawns with only a backpack and other things on

**Comment by Ralta** [comment]
Yeah I mean I know it creates the mob then equips it, but I was under the impression it would only create a human mob and equip them regardless of what the player's species was before. 

I know if you attempt to equip a Vox with ERT gear it won't work because half the sprites are missing, but this shouldn't matter for a mid-round ERT created using 'Create Antagonist'. If we have other species appearing as ERT members, then we have other problems...

**Comment by FlattestGuitar** [comment]
I think the issue is more non-humans being allowed to be on ERTs. That definitely needs to be changed.
